show/main.py

```diff
@@ -567,6 +567,16 @@ def get_interface_bind_to_vrf(config_db, vrf_name):
                     data.append(interface)
     return data
 
+def get_intf_master(interface_name):
```

Comment:
Why to define new function?. This is already available as [get_if_master ](https://github.com/Azure/sonic-utilities/blob/master/show/main.py#L1353)